.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,19 @@
+name: Auto-merge
+
+on:
+  pull_request
+
+jobs:
+  automerge-dependencies:
+    name: Automerge dependencies
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Enable auto-merge for the PR
+        if: >
+          # Auto merge `@wordpress/` dependency PRs
+          startsWith( ${{ github.event.pull_request.title }}, 'Bump @wordpress' )
```

Comment:
We can update this if condition as a means to whitelist auto-merging for other dependency updates.

Reply:
At the moment this won't do anything because the `@wordpress` dependencies are excluded, right?

Reply:
Yep, they're ignored in the Dependabot config.